lib/generators/spree_avatax_official/install/templates/config/initializers/spree_avatax_official.rb

```diff
@@ -1,3 +1,7 @@
+module SpreeAvataxOfficial
+  Config = SpreeAvataxOfficial::Configuration.new
```

Comment:
the same here: https://github.com/spree-contrib/spree_avatax_official/pull/170/files#r761694331

Reply:
Without that there was failing spec:
![image](https://user-images.githubusercontent.com/13260586/144570179-4dc7a464-c001-4c60-a422-07722fe9cbb0.png)